1.0/en/0x10-C04-Infrastructure.md

```diff
@@ -115,6 +115,11 @@ Secure distributed AI deployments including edge computing, federated learning,
 | **4.8.2** | **Verify that** edge devices implement secure boot with verified signatures and rollback protection to prevent firmware downgrade attacks. | 2 | D/V |
 | **4.8.3** | **Verify that** distributed AI coordination uses Byzantine fault-tolerant consensus mechanisms with participant validation and malicious node detection. | 3 | D/V |
 | **4.8.4** | **Verify that** edge-to-cloud communication supports bandwidth throttling, data compression, and secure offline operation with encrypted local storage. | 3 | D/V |
+| **4.8.5** | **Verify that** models packaged within client or edge applications (e.g., mobile, IoT, embedded) are encrypted or obfuscated at rest, and decrypted only within a trusted runtime environment. | 2 | D/V |
```

Comment:
Can we drop "obfuscated"? That's typically reversible with effort, encryption not so much